frontend/src/components/SummaryView/index.jsx

```diff
@@ -1,11 +1,200 @@
-import React from 'react';
-import './_index.scss';
+import React, { useEffect, useState, useHasChanged } from 'react';
+import PropTypes from 'prop-types';
+
+import * as d3 from 'd3';
+import _ from 'lodash';
+
+import { COLORS, getCategoryLevels, getRangeWithValues, getAverages } from '../../utils/attributes';
+import { Checkbox, FormControl, FormControlLabel, FormGroup, IconButton, InputLabel, Select, MenuItem, Input } from '@mui/material';
+import FilterAltIcon from '@mui/icons-material/FilterAlt';
+import CloseOutlined from '@mui/icons-material/CloseOutlined';
+
+function SummaryView({ attributeTypes, data }) {
+  const [yAxisAttr, setXAxis] = useState(attributeTypes.quantitative[1]);
```

Comment:
Since these are all coming via props (and summary chart wont have its own filter panel), we can just have these as regular variables instead of setting them to states, no?
Also we're calling the Summary view inside the Overview component right?

Reply:
Yes, you're totally right. Also, I assumed we would put SummaryView inside Dashboard, but it makes more sense to have it inside Overview. I'll update the PR shortly!